ci/env/generate.go

```diff
@@ -62,6 +61,66 @@ func GenerateEnvFile() error {
 	return writeEnvVars(vars)
 }
 
+func isTag() bool {
+	return os.Getenv("BUILD_TAG") != ""
+}
+
+func isSnapshot() bool {
+	return os.Getenv("BUILD_BRANCH") == "master" && !isTag()
+}
+
+func isPr() bool {
```

Comment:
[Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". ](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
func isPR() bool {
```